webapp/src/tutorialValidator.ts

```diff
@@ -0,0 +1,112 @@
+/// <reference path="../../built/pxtlib.d.ts" />
```

Comment:
this file should be moved into pxtlib so that it can be run in our CLI or web worker. Probably near https://github.com/microsoft/pxt/blob/master/pxtlib/tutorial.ts